lib/content-services/src/lib/content-metadata/components/content-metadata/content-metadata.component.spec.ts

```diff
@@ -1484,6 +1486,37 @@ describe('ContentMetadataComponent', () => {
             const noEditableTagsContainer = fixture.debugElement.query(By.css('div.adf-metadata-properties-tags'));
             expect(noEditableTagsContainer).toBeNull();
         });
+
+        it('should show existing tags when editing another panel and viewing tags panel', async () => {
+            component.displayTags = true;
+            component.readOnly = false;
+            fixture.detectChanges();
+            spyOn(tagService, 'getTagsByNodeId').and.returnValue(of(tagPaging));
+
+            component.ngOnChanges({
```

Comment:
in next line you are calling detectChanges so probably no need to call ngOnChanges manually as it will be executed automatically by detectChanges if input of component has been changed

Reply:
ngOnChanges needed here, removed detectChanges